tests/test_enterprise/management/test_backfill_ecu_table_user_foreign_key.py

```diff
@@ -0,0 +1,114 @@
+"""
+Tests for the Django management command `backfill_ecu_table_user_foreign_key`.
+"""
+
+from unittest.mock import patch
+
+import ddt
+import factory
+from pytest import mark
+
+from django.contrib import auth
+from django.core.management import call_command
+from django.db import DatabaseError
+from django.db.models import signals
+from django.test import TestCase
+
+from enterprise.models import EnterpriseCustomer, EnterpriseCustomerUser
+from test_utils import factories
+
+EXCEPTION = "DUMMY_TRACE_BACK"
+
+User = auth.get_user_model()
+
+
+@mark.django_db
+@ddt.ddt
+class CreateEnterpriseCourseEnrollmentCommandTests(TestCase):
```

Comment:
It would be good to have a test case with an ECU instance that has `linked=False` so we can ensure that the command really is hitting all the rows of the table.